tests/urlpatterns/path_urls.py

```diff
@@ -1,8 +1,17 @@
 from django.urls import include, path, re_path
+from django.utils.translation import gettext_lazy as _
 
 from . import views
 
 urlpatterns = [
+    path(
+        _("test/"),
+        include(
+            [
+                path("child/", lambda request: None, name="child"),
+            ]
+        ),
+    ),
```

Comment:
I'd do something simpler, like this...:
```py
    path(_("lazy-route/"), include("urlpatterns.more_urls")),
```
... and place it near the other miscellaneous complex examples near the bottom.